docs/deployment/private-ddn/index.mdx

```diff
@@ -1,30 +1,53 @@
 ---
-sidebar_position: 6
+sidebar_position: 1
 sidebar_label: Hasura DDN
-description: "Learn how to get started with Hasura DDN and your GraphQL API."
+description:
+  "Get started with Hasura Private DDN to build secure, scalable, and customizable GraphQL APIs on your own
+  infrastructure or in the cloud."
+
 keywords:
   - hasura ddn
-  - graphql api
-  - quickstart
-  - getting started
-  - guide
+  - private ddn
+  - byoc
+  - vpc
+  - enterprise
+  - data plane
 ---
 
 # Hasura Private DDN
 
-To deploy a supergraph to Hasura Private DDN, you first need to have a Data Plane created. When you onboard as a
-customer, a Data Plane would have been provisioned for you by the Hasura team and the details will be shared with you.
+## Introduction
+
+Hasura Private DDN allows you to provision private Data Planes — either on your own infrastructure, or in the cloud — to
```

Comment:
Should we mention about provisioning in the cloud hosted and managed by Hasura?

Reply:
Done!